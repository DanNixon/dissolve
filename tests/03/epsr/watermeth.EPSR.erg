 -0.171972E+02  0.631121E+01  0.718636E-03  0.000000E+00  0.100000E+00  0.174890E-02  0.123377E-02  0.000000E+00  0.000000E+00  0.000000E+00
