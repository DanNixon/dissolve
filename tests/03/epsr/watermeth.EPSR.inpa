           1           0           5   0.00000000       0.00000000       0.00000000       0.00000000    
           1
         120   2.00000009E-03   1.99999996E-02
   0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000    
  0.899999976       1.00000000               1
           1           1           1
   9.7340501922641806E-002  0.99985431679119607        3.7547593200855454E-004  -6.6739828154373563E-004   1.1264220965951984E-003  -5.8274129341113763E-004   3.7547539251486502E-004  -2.4192700452203145E-004   8.6003439744703641E-004  -1.4515550890784462E-003   7.5094742853962107E-004  -4.8385158028735429E-004  -7.6434820765572799E-004   2.5801131883777151E-003  -1.3347983257879664E-003   8.6003738105009081E-004  -2.1773298316233418E-003   2.2528381813529566E-003  -1.4515526306859701E-003  -5.8274007847517362E-004   7.5094610542648628E-004  -2.4192689461063636E-004
           1           1           2
 -0.25087464825204714        3.7547223417865218E-004  0.99903229605476973        1.7200727739561021E-003  -2.9031100122683053E-003   1.5018960196924126E-003  -9.6770439747749504E-004   6.2351249943184170E-004  -2.2165617449161476E-003   3.7410729203086533E-003  -1.9354077972929325E-003   1.2470244655868445E-003   1.9699489852427559E-003  -6.6496865134432811E-003   3.4401492882472052E-003  -2.2165624696988710E-003   5.6116119430463587E-003  -5.8062205229403151E-003   3.7410768270939917E-003   1.5018959057076097E-003  -1.9354037565314983E-003   6.2351366845459920E-004
           1           1           3
  0.44592474686203837       -6.6739584223017470E-004   1.7200756791877396E-003  0.99694260067856033        5.1602193692440760E-003  -2.6695886234469669E-003   1.7200736174868821E-003  -1.1082837666496535E-003   3.9398956160505922E-003  -6.6496883412382607E-003   3.4401467654418608E-003  -2.2165635920615196E-003  -3.5015465593401171E-003   1.1819691994105111E-002  -6.1147973910824806E-003   3.9398950441825696E-003  -9.9745282960119093E-003   1.0320442549072998E-002  -6.6496865568247340E-003  -2.6695855813915442E-003   3.4401481096201060E-003  -1.1082839268978842E-003
           1           1           4
 -0.75262373857497800        1.1264222328223048E-003  -2.9031098568935332E-003   5.1602201620264158E-003  0.99129067085276168        4.5056821829031792E-003  -2.9031119325330108E-003   1.8705373127683538E-003  -6.6496856894582602E-003   1.1223223314411746E-002  -5.8062198746603750E-003   3.7410705313051327E-003   5.9098467281584839E-003  -1.9949064717612160E-002   1.0320444483616740E-002  -6.6496866837561187E-003   1.6834835972343049E-002  -1.7418657473787878E-002   1.1223222099117818E-002   4.5056823191506716E-003  -5.8062168208952939E-003   1.8705392518001739E-003
           1           1           5
  0.38936221379231023       -5.8274145805808409E-004   1.5018922963312841E-003  -2.6695864457362798E-003   4.5056811705002451E-003  0.99766903152643915        1.5018939926903642E-003  -9.6770104588657943E-004   3.4401459584566785E-003  -5.8062172215645764E-003   3.0037893192727989E-003  -1.9354049286928016E-003  -3.0573975356696314E-003   1.0320443434960217E-002  -5.3391752292994048E-003   3.4401477385532400E-003  -8.7093313046855969E-003   9.0113676785725481E-003  -5.8062204910655823E-003  -2.3309659281234986E-003   3.0037872073344265E-003  -9.6770143685581862E-004
           1           1           6
 -0.25087457538363378        3.7547297060441489E-004  -9.6770197652245877E-004   1.7200737829729464E-003  -2.9031076347137627E-003   1.5018927786851115E-003  0.99903229696488283        6.2351065942783121E-004  -2.2165602007780910E-003   3.7410749559868093E-003  -1.9354085319485330E-003   1.2470249599101225E-003   1.9699490111285333E-003  -6.6496843844691539E-003   3.4401470476042662E-003  -2.2165615398081623E-003   5.6116111649919297E-003  -5.8062184893939205E-003   3.7410736911667971E-003   1.5018933180024421E-003  -1.9354064746911110E-003   6.2351386584404004E-004
           1           2           2
  0.16164408097292174       -2.4192593404651224E-004   6.2351390849722007E-004  -1.1082820495741843E-003   1.8705408914751387E-003  -9.6770237030507032E-004   6.2351380739704551E-004  0.99959825773904953        1.4281819592633607E-003  -2.4104556815682052E-003   1.2470218767549445E-003  -8.0348994007936006E-004  -1.2692831761313323E-003   4.2845394586607894E-003  -2.2165584507003338E-003   1.4281813642228020E-003  -3.6156846057999866E-003   3.7410737851002978E-003  -2.4104596417279280E-003  -9.6770183135003890E-004   1.2470256199405771E-003  -4.0174289155086966E-004
           1           2           3
 -0.57463810724354314        8.6003666101619751E-004  -2.2165629528479737E-003   3.9398985873267036E-003  -6.6496883210822736E-003   3.4401496496815295E-003  -2.2165616069257121E-003   1.4281806159708518E-003  0.99492287643267130        8.5690792091377300E-003  -4.4331257310733004E-003   2.8563612294298900E-003   4.5122477891968030E-003  -1.5231368599786511E-002   7.8797936141436487E-003  -5.0771229836275005E-003   1.2853617625785928E-002  -1.3299375370817526E-002   8.5690776036038852E-003   3.4401496723628919E-003  -4.4331266858908865E-003   1.4281783222246671E-003
           1           2           4
  0.96986400257119554       -1.4515565228413411E-003   3.7410761067411201E-003  -6.6496873494943419E-003   1.1223221045998858E-002  -5.8062212321815495E-003   3.7410756245315662E-003  -2.4104561038468843E-003   8.5690779029066501E-003  0.98553725925205915        7.4821481608777773E-003  -4.8209111912340102E-003  -7.6156838546792838E-003   2.5707236624916276E-002  -1.3299378181383222E-002   8.5690781798465655E-003  -2.1694105211586732E-002   2.2446445207282579E-002  -1.4462735183348919E-002  -5.8062182270680806E-003   7.4821541002352892E-003  -2.4104558621115394E-003
           1           2           5
 -0.50174918885880604        7.5094822157868908E-004  -1.9354055153308130E-003   3.4401455541399352E-003  -5.8062182334019654E-003   3.0037846758008023E-003  -1.9354044964873160E-003   1.2470256717182382E-003  -4.4331263026001514E-003   7.4821474217136481E-003  0.99612918554965302        2.4940503698228323E-003   3.9398931328395630E-003  -1.3299375186672605E-002   6.8802954736768233E-003  -4.4331268267081671E-003   1.1223222431827041E-002  -1.1612441235644724E-002   7.4821471998414973E-003   3.0037872950079824E-003  -3.8708118573580015E-003   1.2470263163090172E-003
           1           2           6
  0.32328797875284876       -4.8385002139583315E-004   1.2470219569779032E-003  -2.2165597669817649E-003   3.7410727830675464E-003  -1.9354078829807208E-003   1.2470237424476430E-003  -8.0348379579781048E-004   2.8563612305757017E-003  -4.8209121506947051E-003   2.4940498033839344E-003  0.99839302855316059       -2.5385617801338537E-003   8.5690808806349692E-003  -4.4331279304708230E-003   2.8563602610408678E-003  -7.2313690185331425E-003   7.4821505207681343E-003  -4.8209127039133748E-003  -1.9354077454839186E-003   2.4940506504674289E-003  -8.0348650116201083E-004
           1           3           3
  0.51070410217232609       -7.6435110607978702E-004   1.9699491909867936E-003  -3.5015489720892430E-003   5.9098468600798336E-003  -3.0573986601395721E-003   1.9699497456310343E-003  -1.2692826754488284E-003   4.5122441432655204E-003  -7.6156879357767394E-003   3.9398967652924570E-003  -2.5385653143539912E-003  0.99598978680404326        1.3536734325378916E-002  -7.0030934348399854E-003   4.5122456261202908E-003  -1.1423530109890707E-002   1.1819692617720354E-002  -7.6156854918133909E-003  -3.0573933367591321E-003   3.9398964615595458E-003  -1.2692820531102326E-003
           1           3           4
  -1.7239144957004315        2.5801106128765382E-003  -6.6496866312061422E-003   1.1819691712329642E-002  -1.9949061938518319E-002   1.0320442802442206E-002  -6.6496878354103655E-003   4.2845397743766734E-003  -1.5231369736860416E-002   2.5707239007920574E-002  -1.3299374203635252E-002   8.5690821982127643E-003   1.3536733084177755E-002  0.95430588409261152        2.3639385451674589E-002  -1.5231371011190474E-002   3.8560857918319924E-002  -3.9898126490168433E-002   2.5707238188218570E-002   1.0320444361098803E-002  -1.3299374364022128E-002   4.2845419171859593E-003
           1           3           5
  0.89184944810812827       -1.3347893782983988E-003   3.4401467843672602E-003  -6.1147959868955559E-003   1.0320444263457063E-002  -5.3391773510099187E-003   3.4401483648298647E-003  -2.2165644662384392E-003   7.8797939088520209E-003  -1.3299379084474995E-002   6.8802972391632904E-003  -4.4331233496098616E-003  -7.0030947775993431E-003   2.3639384811788424E-002  0.98777040963456697        7.8797910557939950E-003  -1.9949062983395910E-002   2.0640883664072511E-002  -1.3299378570559873E-002  -5.3391771208357508E-003   6.8802955443576118E-003  -2.2165597264617344E-003
           1           3           6
 -0.57463812478545417        8.6003772052208895E-004  -2.2165608415780887E-003   3.9398950281242994E-003  -6.6496855429062987E-003   3.4401476886284973E-003  -2.2165626562105326E-003   1.4281832610676247E-003  -5.0771244212948511E-003   8.5690809055498571E-003  -4.4331258310219184E-003   2.8563593990086609E-003   4.5122449830655945E-003  -1.5231370113175486E-002   7.8797943000450590E-003  0.99492287442670369        1.2853619485020560E-002  -1.3299376837066066E-002   8.5690801074090901E-003   3.4401464799326655E-003  -4.4331233467523904E-003   1.4281787714969492E-003
           1           4           4
   1.4547959390907421       -2.1773314676778463E-003   5.6116132744104064E-003  -9.9745321957347959E-003   1.6834833981983852E-002  -8.7093312387243899E-003   5.6116102450872027E-003  -3.6156833193245310E-003   1.2853619821389706E-002  -2.1694107601122406E-002   1.1223225094155668E-002  -7.2313672440492919E-003  -1.1423529625543263E-002   3.8560854443858816E-002  -1.9949064244535127E-002   1.2853616495536119E-002  0.96745883926783272        3.3669669403013296E-002  -2.1694104677840359E-002  -8.7093336812516867E-003   1.1223222632685085E-002  -3.6156833077103964E-003
           1           4           5
  -1.5052475172496858        2.2528425007501160E-003  -5.8062201142554315E-003   1.0320442300635845E-002  -1.7418659738439816E-002   9.0113646579451905E-003  -5.8062203745714511E-003   3.7410728675161811E-003  -1.3299375063586312E-002   2.2446447378961534E-002  -1.1612437549007227E-002   7.4821495734004794E-003   1.1819692563197931E-002  -3.9898125676914267E-002   2.0640885554201387E-002  -1.3299371805777302E-002   3.3669671746090285E-002  0.96516268516518389        2.2446447565784889E-002   9.0113665037631525E-003  -1.1612437866090817E-002   3.7410735204702278E-003
           1           4           6
  0.96986398929624273       -1.4515544893210816E-003   3.7410770869639792E-003  -6.6496882370643490E-003   1.1223223429991852E-002  -5.8062193758828809E-003   3.7410763616881667E-003  -2.4104561650611662E-003   8.5690802700112163E-003  -1.4462737135859226E-002   7.4821441430791913E-003  -4.8209135663853635E-003  -7.6156869236800023E-003   2.5707239730013296E-002  -1.3299374071468322E-002   8.5690783976478391E-003  -2.1694104313436831E-002   2.2446446204802305E-002  0.98553726123302232       -5.8062193163826367E-003   7.4821535461072576E-003  -2.4104544473755180E-003
           1           5           5
  0.38936222023427780       -5.8274077529891642E-004   1.5018973164123881E-003  -2.6695880804907569E-003   4.5056823081358359E-003  -2.3309708629200253E-003   1.5018910387936783E-003  -9.6770206393486801E-004   3.4401483613432527E-003  -5.8062204087048885E-003   3.0037860862031488E-003  -1.9354076573454898E-003  -3.0573980888508640E-003   1.0320441180860226E-002  -5.3391737568301073E-003   3.4401496837284809E-003  -8.7093308135733506E-003   9.0113617843503423E-003  -5.8062195669241036E-003  0.99766903258139450        3.0037898163135929E-003  -9.6770277557174276E-004
           1           5           6
 -0.50174920360021213        7.5094783388978228E-004  -1.9354081305573390E-003   3.4401475104665030E-003  -5.8062197182283774E-003   3.0037869334500959E-003  -1.9354079457570977E-003   1.2470250228361131E-003  -4.4331219391271718E-003   7.4821500113096484E-003  -3.8708133705961196E-003   2.4940488659841308E-003   3.9398967719831510E-003  -1.3299373498138457E-002   6.8802973412139265E-003  -4.4331241651487559E-003   1.1223221288242918E-002  -1.1612436706015566E-002   7.4821471674370327E-003   3.0037887930458337E-003  0.99612918518194793        1.2470249476923818E-003
           1           6           6
  0.16164397931478980       -2.4192242333963824E-004   6.2351208779373095E-004  -1.1082792476025053E-003   1.8705396927821743E-003  -9.6770488372323251E-004   6.2351128026029814E-004  -4.0174112602752089E-004   1.4281790433929866E-003  -2.4104543548310350E-003   1.2470252898404176E-003  -8.0348347436999664E-004  -1.2692788770060062E-003   4.2845402546305362E-003  -2.2165604788409619E-003   1.4281814214284335E-003  -3.6156860544386523E-003   3.7410750923169002E-003  -2.4104566764112947E-003  -9.6770395958215949E-004   1.2470260375604932E-003  0.99959825781653655     
