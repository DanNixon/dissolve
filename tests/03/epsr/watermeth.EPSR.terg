 -0.687641E+01  0.000000E+00 -0.687641E+01
 OW  OW 
  0.387271E+02  0.000000E+00  0.387271E+02
 OW  HW 
 -0.799775E+02  0.000000E+00 -0.799775E+02
 OW  CT 
  0.956992E+01  0.000000E+00  0.956992E+01
 OW  HC 
 -0.289410E+02  0.000000E+00 -0.289410E+02
 OW  OH 
  0.460951E+02  0.000000E+00  0.460951E+02
 OW  HO 
 -0.291932E+02  0.000000E+00 -0.291932E+02
 HW  HW 
  0.389276E+02  0.000000E+00  0.389276E+02
 HW  CT 
 -0.101777E+02  0.000000E+00 -0.101777E+02
 HW  HC 
  0.292718E+02  0.000000E+00  0.292718E+02
 HW  OH 
 -0.484491E+02  0.000000E+00 -0.484491E+02
 HW  HO 
  0.295818E+02  0.000000E+00  0.295818E+02
 CT  CT 
  0.133460E+00  0.000000E+00  0.133460E+00
 CT  HC 
 -0.462519E+01  0.000000E+00 -0.462519E+01
 CT  OH 
  0.620516E+01  0.000000E+00  0.620516E+01
 CT  HO 
 -0.448254E+01  0.000000E+00 -0.448254E+01
 HC  HC 
  0.674470E+01  0.000000E+00  0.674470E+01
 HC  OH 
 -0.210380E+02  0.000000E+00 -0.210380E+02
 HC  HO 
  0.130501E+02  0.000000E+00  0.130501E+02
 OH  OH 
  0.158117E+02  0.000000E+00  0.158117E+02
 OH  HO 
 -0.204754E+02  0.000000E+00 -0.204754E+02
 HO  HO 
  0.636466E+01  0.000000E+00  0.636466E+01
