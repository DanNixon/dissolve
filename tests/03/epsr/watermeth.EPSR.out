
  EPSR version 25: 2016-02-24      

  Trial no.     1      Iteration no.     1      No. of configurations in sum     1
  Chi-square  0.718636E-03 R-space coeff.:  0.174890E-02 Abs. empirical energy:   0.000000E+00
  Quality factor  0.123377E-02 Control level, baseline and threshold:  0.000000E+00  0.000000E+00  0.500000E+00
  Pair energies: repulsive -0.171954E+02 harmonic  0.114905E+02 per molecule
  Tethered energy:  0.000000E+00 per molecule
  Three-body energy:  0.000000E+00 per molecule
  Three-body pressure:  0.000000E+00 kbar
  No. of .erg lines:       0 Average gradient of quality factor:  0.000000E+00
  Sizefactor    1.00000 Number density  0.10000E+00
  Net charge =   -0.00001
  R-factor for data file  1:  0.7186E-03
  Noise in average g(r)  0.839800E+02
  Intermolecular energy and U_pol -0.171972E+02  0.000000E+00 kJ/mole
  Intermolecular pressure  0.631121E+01 kbar
  Average dipole moment and std.  0.163824E+01  0.618284E+00 D
  Av. intermolecular energy  0.000000E+00+- 0.000000E+00 kJ/mole
  Av. intermolecular pressure  0.000000E+00+- 0.000000E+00 kbar
  Number of rejects in MOLFIT:-
    9000    6000       0       0
    7232    1728       0       0
